### ISSUE TITLE
Кеширование данных

### ISSUE THREAD

**Issue by bublikOff** [bastshoes/yandex_weather#10]
Было бы интересно, если компонента кешировала результат запросов ... и не запрашивала бы данные с яндекса при каждом запросе. При перезапусках во время тестирования автоматизации и тп быстро выжирает кол-во запросов к API

**Comment by sshalyminov** [comment]
Да, было бы полезно. И, как добавка, - добавить опцию отключения этих автоматических запросов, отдавать в систему кешированные данные, и добавить сервис для ручного вызова запроса. Тогда можно было бы настроить автоматизацию вызова обновления с нужным интервалом (скажем, когда количество часов в сутки кратно 4 или любой другой цифре), а в остальное время смотреть результат последнего вызова. А по дефолту оставить то, что есть сейчас - для обратной совместимости и для тех, кто поленится лезть изучать, как правильно настроить такое ручное обновление.
